main.go

```diff
@@ -223,16 +222,9 @@ func getExportData(fs *fullstory.Client, bundleID int) (fullstory.ExportData, er
 		log.Printf("Failed to fetch export data for Bundle %d: %s", bundleID, err)
 
 		fsErr = err
-		retryAfterDuration := defaultRetryAfterDuration
-		if statusError, ok := err.(fullstory.StatusError); ok {
-			// If the status code is NOT 429 and the code is below 500 we will not attempt to retry
-			if statusError.StatusCode != http.StatusTooManyRequests && statusError.StatusCode < 500 {
-				break
-			}
-
-			if retryAfter, err := strconv.Atoi(statusError.RetryAfter); err == nil {
-				retryAfterDuration = time.Duration(retryAfter) * time.Second
-			}
+		doRetry, retryAfterDuration := getRetryInfo(err)
+		if !doRetry {
+			break
 		}
 
 		log.Printf("Attempt #%d failed. Retrying after %s\n", r, retryAfterDuration)
```

Comment:
This can be a TODO; should we maybe log the error here instead of assuming that all errors from one set of `ExportData` calls will always be the same?

Reply:
The `err` is logged separately on line 222 above, and then the very last one to occur is returned below. Does that address your concern? Ultimately, if all attempts fail, the last attempt will be logged twice, once on line 222 and then again wherever this bubbles up in the main algorithm.

Reply:
Oops! Sorry, I totally missed the log on line 222. LGTM.